src/test/java/core/basesyntax/db/StorageImplTest.java

```diff
@@ -0,0 +1,109 @@
+package core.basesyntax.db;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class StorageImplTest {
+    private Map<String, Integer> map;
+    private Storage storage;
+
+    @BeforeEach
+    void setUp() {
+        map = new LinkedHashMap<>();
+        storage = new StorageImpl(map);
+    }
+
+    @Test
+    void getFruit_notNull_notOk() {

```

Comment:
do not use "notOk" give more clear description whats not ok.

```suggestion
void getFruit_notNull_throwsRuntimeException() {
```

apply to all tests

Reply:
also, test name specifies "notNull" but test input is ` storage.getFruit(null);`. Please fix a name of the test